api_tests/harness.js

```diff
@@ -177,23 +182,24 @@ async function startLedgerQueryService(name, lqs_config) {
             ],
         }
     );
-    subprocesses.push(subprocess);
 }
 
 // ********************************** //
 // Start services, run test, shutdown //
 // ********************************** //
 
-describe("Starting services", async function() {
+describe("🛠", async function() {
```

Comment:
😄

Reply:
Is that better? I think it renders ok:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/300805/50123371-76157f80-02b4-11e9-815d-f2102a85afea.png">